recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
ok, so in this sense I still need to think. Meanwhile the person can provide an /etc overlay with links to /dev/null, that is the mask of services that shouldn't be started.

Problem is how to define which services are in the parent image, right? My idea is that on the host, only a handful services are to be visible, those we could even document: connman, swupd, etc.

As for  the appfw generated, they are in /run, the /run from host shouldn't be exposed to the child containers.